sources/common/toddl.go

```diff
@@ -234,6 +235,21 @@ func cvtForeignKeys(conv *internal.Conv, spTableName string, srcTableId string,
 	return spKeys
 }
 
+func cvtCheckConstraint(conv *internal.Conv, srcKeys []schema.CheckConstraint) []ddl.CheckConstraint {
+	var spcks []ddl.CheckConstraint
+
+	for _, cks := range srcKeys {
+		spcks = append(spcks, ddl.CheckConstraint{
+			Id:   cks.Id,
+			Name: internal.ToSpannerCheckConstraintName(conv, cks.Name),
+			Expr: cks.Expr,
+		})
+
+	}
+
+	return internal.GetSpannerValidExpression(spcks)
```

Comment:
This API creates and deletes a spanner database. It shouldn't be called on a per table basis but rather once for all tables for the following reasons:
1. Both create and delete db are expensive and long running operations
2. Table migration is a parallelized op and there is a limit on the number of times create and delete tables can be called. Further the db created is always named the same so this can cause race conditions

Reply:
removed this API call.